integration-tests/test/rnode.py

```diff
@@ -512,7 +512,8 @@ def make_bootstrap_node(
     number_of_active_validators: int = 10,
     epoch_length: int = 10000,
     quarantine_length: int = 50000,
-    min_phlo_price: int = 1
+    min_phlo_price: int = 1,
+    shard_id: str = ''
```

Comment:
It should not be possible to start the node with empty shardId. Do we have this check?

Reply:
@tgrospic, yes, we have this check https://github.com/rchain/rchain/blob/45b83dfe614f30e4705fc1649a6266da3f4f9e95/casper/src/main/scala/coop/rchain/casper/Validate.scala#L131

This is the reason why I provided `shard_id = 'test` in integration tests. I can remove the default empty string if needed.